iot-hub/Quickstarts/read-d2c-messages/readme.md

```diff
@@ -1,15 +1,27 @@
 # Notes
 
-## Pre-requisite ##
-- **Node.js version: 6.x or higher.** 
-- We would **still encourage you** to install the latest available LTS version at any given time from https://nodejs.org. **It is a good practice to always install the latest available LTS version of node.js.**
-- Installing node.js on **Windows or macOS** is very simple with available installers on the [node.js website](https://nodejs.org). If you are using a **linux based OS**, then you can find easy to follow, one step installation instructions over [here](https://nodejs.org/en/download/package-manager/).
+This sample demonstrates how to use the Microsoft Azure Event Hubs Client for JavaScript to 
+read messages sent from a device by using the built-in event hubs that exists by default for
+every Iot Hub instance. 
+
+If you have access to the Event Hubs-compatible endpoint, either via the Azure portal or
+by using the Azure CLI, you can skip the parts in this sample that converts the Iot Hub
+connection string to an Event Hubs compatible one.
+
```

Comment:
another small suggestion: perhaps we should call out that a user must add their iothub connection string to the iotHubConnectionString variable in the sample as a prereq?

Reply:
addressed in #38